lib/client-api/src/story_store.ts

```diff
@@ -33,25 +43,45 @@ const toExtracted = obj =>
     return Object.assign(acc, { [key]: value });
   }, {});
 
+interface Selection {
+  storyId: string;
+  viewMode: string;
+}
+
 export default class StoryStore extends EventEmitter {
-  constructor(params) {
+  _error?: ErrorLike;
+
+  _channel: Channel;
+
+  _data: StoreData;
+
+  _legacyData?: LegacyData;
```

Comment:
My guess is that you were trying to fix naming from `_legacydata` to `_legacyData` but somehow not finished or discarded chaning?

Reply:
I'm fearful of changing too much, this SHOULd be an internal api, but I am aware of some integrations hooking into this, and preferably I would like to change it at a later point at the 6.0 release instead.

Reply:
@ndelangen what's `_legacyData`?! Seems unused throughout the codebase?